### PR TITLE
Add Vanta, DoIt, ProsperOps, nOps, Drata, Usage AI, Prisma Cloud, Vectra, Turbot, Sumo Logic, Mackerel, Canonical, Archera, Flexera AWS account IDs

### PR DESCRIPTION
**Pull request by christophetd**
The ProsperOps and UsageAI AWS accounts are accessible to anyone, after login with a free account (screenshots below)

image

image

### CODE REVIEW

File: accounts.yaml [hunk 1]
```diff
@@ -448,4 +457,47 @@
...
   source: 'https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_sns.html'
   accounts: ['242987662583','118283430703','144182107116','934957504740','107430051933','973841112453','965013871422','506403581195','436163563069','378365507264','383009515534','646602203151','741172661024','464168911255','476419727788','457615622431','926826061926','955633302743','430639793359','002991280229','003033775354','552740612889','030935290150','188461706213','445632894446','143972945659','129086577509','225965583551','595653072700','529900636122']
-  
+- name: 'Vanta'
+  type: 'aws'
+  source: 'https://help.vanta.com/hc/en-us/articles/8451925240980-Connecting-Vanta-AWS-Organization'
+  accounts: ['956993596390']
+- name: 'Wiz'
+  type: 'aws'
+  source: 'https://github.com/rupertbg/aws-public-account-ids/blob/master/accounts.json#L500C10-L500C22'
```
Comment (0xdabbad00) [inline accounts.yaml:466]: I want to try to use references from the vendors. For whatever reason Wiz does not make their account IDs public.

Comment (christophetd) [reply accounts.yaml:466]: How about we link to the (private) Wiz docs? At least it's the vendor reference, even if only customers can access it (it's effectively public anyway)

File: accounts.yaml [hunk 2]
```diff
@@ -448,4 +457,43 @@
...
+  type: 'aws'
+  source: 'https://github.com/drata/terraform-aws-drata-autopilot-role/blob/f774d423a62df3a3dd03008a68c3b3d70b95be33/variables.tf#L1-L5'
+  accounts: ['269135526815']
+- name: 'DoIt'
+  type: 'aws'
+  source: 'https://help.doit.com/docs/flexsave/aws/standalone/required-policies'
+  accounts: ['068664126052']
+- name: 'ProsperOps'
+  type: 'aws'
+  source: 'https://console.prosperops.com/onboarding'
```
Comment (new23d) [inline accounts.yaml:474]: requires login, so couldn't verify

File: accounts.yaml [hunk 3]
```diff
@@ -448,4 +457,43 @@
...
+  accounts: ['205499583182']
+- name: 'nOps'
+  type: 'aws'
+  source: 'https://docs.nops.io/en/articles/6818153-aws-setup-permissions-sharesave-auto-pilot-risk-free-commitment-management'
```
Comment (new23d) [inline accounts.yaml:478]: i'm getting a 404 on this link, so couldn't verify

File: accounts.yaml [hunk 4]
```diff
@@ -448,4 +457,43 @@
...
+  accounts: ['727378841472']
+- name: 'Usage AI'
+  type: 'aws'
+  source: 'https://cloudopt.usage.ai/onboard'
```
Comment (new23d) [inline accounts.yaml:482]: requires login, so couldn't verify

File: accounts.yaml [hunk 5]
```diff
@@ -448,4 +457,43 @@
...
+  accounts: ['826182721854']
+- name: 'Mackerel'
+  type: 'aws'
+  source: 'https://mackerel.io/docs/entry/integrations/aws#:~:text=How%20to%20configure%20an%20IAM%20role'
+  accounts: ['217452466226']
+- name: 'Canonical'
+  type: 'aws'
+  source: 'https://ubuntu.com/blog/time-to-prepare-for-ubuntu-18-04-lts-end-of-standard-support-on-31-may-2023-options-for-aws-users#:~:text=EC2%20describe%2Dimages%3A-,aws%20ec2%20describe%2Dimages,-%2D%2Doutput%20text%20%2D%2Downer'
```
Comment (new23d) [inline accounts.yaml:490]: `https://ubuntu.com/server/docs/cloud-images/amazon-ec2` is a simpler, more authoritative source

**Review by new23d — CHANGES_REQUESTED** [review]

File: accounts.yaml (hunk 3 above)
Comment (christophetd) [reply accounts.yaml:478]: indeed, looks like they changed the URL. The article still shows up if you search for the account ID on Google

image

Will update to their new docs https://help.nops.io/iam-yaml-sharesave.html#sharesave-yaml-file

Comment (christophetd) [inline accounts.yaml:478]:
```suggestion
  source: 'https://help.nops.io/iam-yaml-sharesave.html#sharesave-yaml-file'
```

File: accounts.yaml (hunk 5 above)
Comment (christophetd) [inline accounts.yaml:490]:
```suggestion
  source: 'https://ubuntu.com/server/docs/cloud-images/amazon-ec2'
```

File: accounts.yaml (hunk 2 above)
Comment (new23d) [reply accounts.yaml:474]: screenshot confirms the account id

File: accounts.yaml (hunk 4 above)
Comment (new23d) [reply accounts.yaml:482]: screenshot confirms the account id

**Review by new23d — APPROVED** [review]

**Comment by new23d** [comment]
I have verified all the sources. This PR is good to merge.

**Pull request merged**